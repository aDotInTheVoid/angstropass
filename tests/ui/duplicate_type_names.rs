#[angstropass::langs]
mod langs {
    mod l1 {
        type Foo = i32;
        type Foo = i32;
    }
}

fn main() {}
