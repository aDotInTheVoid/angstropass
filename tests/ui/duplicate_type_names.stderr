error: duplicate definition of Foo
 --> tests/ui/duplicate_type_names.rs:5:9
  |
5 |         type Foo = i32;
  |         ^^^^^^^^^^^^^^^
